api.md

```diff
@@ -2013,10 +2093,10 @@ removes the existing reorder policy for the `conditions` table if it exists.
 ---
 
 
-## alter_job_schedule() :community_function: [](alter_job_schedule)
+## alter_job() :community_function: [](alter_job)
 
 Policy jobs are scheduled to run periodically via a job run in a background
-worker. You can change the schedule using `alter_job_schedule`. To alter an
+worker. You can change the schedule using `alter_job`. To alter an
```

Comment:
This specifically refers to schedule. But we should be able to change any attribute. Need changes to reflect that.